muesli/web/templates/start.pt

```diff
@@ -2,14 +2,28 @@
 <metal:main metal:use-macro="templates('Fragments/main.pt').macros['main']">
   <metal:main metal:fill-slot="content">
 
+<script type="text/javascript" src="http://code.jquery.com/jquery-1.10.0.min.js"></script>
```

Comment:
please remove this here as well :-)

Reply:
This was done to fix an error on my machine, which could not (and still can't) find the jQuery files that were needed.

This line is removed in the newest commit.